how-we-work/all-hands-meeting.html.md

```diff
@@ -1,14 +1,39 @@
 ---
 title: All Hands meeting
 ---
-Every month, we hold an All Hands meeting. These meetings are used to:
-
-- give a general update on the status of Nebulab;
-- introduce new team members;
-- present any important updates to the playbook.
-
-You are strongly encouraged to attend so that you can ask questions and spend some virtual
-bonding time with the entire team, but in case you're not able to, the meetings are recorded.
+Every month, we hold a company-wide meeting, called the All Hands meeting. We use this
+session to provide useful information to every team member, with the intent of being as much
+transparent as possible about the direction of Nebulab, along with its successes, failures
+and next things to do. Having everyone on the same page is our way to try to drive every
+single decision made towards a shared direction. The meeting covers sevaral topics:
+
+- **Welcome.** Here we welcome new team members. Often, there is a presentation of new
+  hires in front of the rest of team so everyone knows some information about them.
+  Here we may also talk about people leaving the company, if that happens.
+- **Strategy.** This topic covers the high-level decisions that we are making to make
+  Nebulab grow. We can talk about new sales initiatives, our hiring process or how we
+  are setting things up for personal and professional growth of individuals. We share
+  such information with everyone both to be aligned and to receive any feedback about
+  the direction.
+- **New projects.** Have we got a new client? Here we can introduce them, talking about
```

Comment:
This is more of a personal opinion, but the question at the beginning has a marketing vibe to it. 😄 I would have put "We introduce new clients".

If you think the question sounds fun, though, you can keep it, but the American English version would be "Do we have new clients?".